app/helpers/application_helper.rb

```diff
@@ -36,9 +36,7 @@ def home_path
   end
 
   def otp_delivery_method_options
-    User.otp_delivery_methods.keys.map do |key|
-      OpenStruct.new(id: key, name: t(".#{key}"))
```

Comment:
The form builder docs [still suggest using OpenStruct](https://govuk-form-builder.netlify.app/form-elements/select/) which is interesting given that rubocop recommends _not_ using it!